ntbs-service/wwwroot/source/Components/CascadingDropdowns.ts

```diff
@@ -12,15 +15,53 @@ const CascadingDropwdowns = Vue.extend({
             return response.data;
           })
           .then((data: any) => {
-            this.$refs["hospitalDropdown"].innerHTML = "<option value=''>Select Hospital</option>";
+            this.$refs["selectFieldHospitalId"].innerHTML = "<option value=''>Select Hospital</option>";
 
             data.forEach((item: any) => {
-              this.$refs["hospitalDropdown"].innerHTML += `<option value="${item.hospitalId}">${item.name}</option>`
+              this.$refs["selectFieldHospitalId"].innerHTML += `<option value="${item.hospitalId}">${item.name}</option>`
             });
           })
           .catch((error: any) => {
               console.log(error.response)
           });
+      },
+      validate: function(property : string) {
+          var shouldValidateFull = this.$props.shouldvalidatefull;
+          var modelName = this.$props.model;
+          // TODO: We need to check whether arrow ('=>') works in IE
+          return (event: Event) => {
+            // Our onBlur validate events happen on input fields
+            const inputField = event.target as HTMLInputElement
+            const newValue = inputField.value;
+            
+            let requestConfig = {
+              url: `${getValidationPath(modelName)}Property`,
+              headers: getHeaders(),
+              params: {
+                "value": newValue,
+                "shouldValidateFull": shouldValidateFull,
+                "key": property
+              }
+            }
+
+            axios.request(requestConfig)
+            .then((response: any) => {
+                var errorMessage = response.data;
+                var hasError = errorMessage != '';
+                this.$refs["errorField" + property].textContent = errorMessage;
+
+                if (hasError) {

```

Comment:
Will try to commonise this bits if needed

Reply:
Don't think it'll be worth it here tbh